plenum/test/view_change/test_no_future_view_change_while_catchup.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+from plenum.common.startable import Mode
+from plenum.test.delayers import icDelay
+from plenum.test.helper import checkViewNoForNodes, waitForViewChange
+from plenum.test.node_catchup.helper import ensure_all_nodes_have_same_data
+from plenum.test.stasher import delay_rules
+from plenum.test.test_node import ensureElectionsDone, checkProtocolInstanceSetup
+from plenum.test.view_change.helper import ensure_view_change
+from plenum.test.waits import expectedPoolViewChangeStartedTimeout
+
+
+@pytest.fixture(params=['starting', 'discovering', 'discovered', 'syncing'])
```

Comment:
Do we really need strings here and lot's of if/else? Would the following do the job?
```
@pytest.fixture(params=[Mode.starting, Mode.discovering...])
def mode(request)
    return request.param
```